tests/lint/unittest_lint.py

```diff
@@ -64,7 +64,7 @@ def fake_home() -> Iterator:
     old_home = os.environ.get(HOME)
     try:
         os.environ[HOME] = folder
-        yield
+        yield folder
```

Comment:
Could you update the typing of this function? `Iterator[str]` should be correct.